api-generation/src/main/scala/ai/h2o/sparkling/api/generation/common/AlgorithmConfigurations.scala

```diff
@@ -97,6 +100,10 @@ class AlgorithmConfigurations extends MultipleAlgorithmsConfiguration {
     val gamFields = Seq(ignoredCols, betaConstraints, gamCols)
     val gbmFields = Seq(monotonicity, calibrationDataFrame, ignoredCols)
     val drfFields = Seq(calibrationDataFrame, ignoredCols)
+    val upliftDrfFields = Seq(
+      ExplicitField("treatment_column", "HasTreatmentCol", "treatment"),
+      ExplicitField("response_column", "HasLabelCol", "label"),
```

Comment:
Why do we need to specify label column explicitly? Isn't uplift DRF just another supervised algorithm?

Reply:
The response column is used the same way as in the supervised algorithm.